artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -1830,6 +1830,27 @@
                </xsd:documentation>
             </xsd:annotation>
          </xsd:element>
+         <xsd:element name="jdbc-lock-acquisition-timeout" type="xsd:int" minOccurs="0" maxOccurs="1">
```

Comment:
There is a lot of JDBC config, is it maybe worth making a JDBC config section and type, so its more contained

Reply:
ignore this, i didn't realise its within a database store type already...helps to read up